inc/pagetop-btn/_scss/_pagetop-btn.scss

```
@@ -41,6 +41,11 @@
 		background-color: transparent;
 		box-shadow: none;
 	}
+	// テーマや他の CSS で `a:hover { text-decoration: underline; color: ...; }` のように
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)